Dockerfile

```diff
@@ -12,24 +12,23 @@
 
 # coreutils is included to use timeout in travis-ci builds
 
-FROM docker.io/ubuntu:xenial-20180417
+FROM docker.io/ubuntu:bionic-20190307
 
-LABEL version="8" \
+LABEL version="9" \
       maintainer="Marcel Metz <mmetz@adrian-broher.net>"
 
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN echo "# don't install documentation inside container\n\
-path-exclude=/usr/share/doc/*\n\
```

Comment:
Why did you drop the default documentation installation location?

Reply:
I remember having this exclude line is a part of bionic docker image. I'll recheck if it present.

Reply:
This line is in a `/etc/dpkg/dpkg.cfg.d/excludes`.